domain/ide/command/folder_handler.go

```diff
@@ -33,23 +33,22 @@ import (
 const DoTrust = "Trust folders and continue"
 const DontTrust = "Don't trust folders"
 
-func HandleFolders(ctx context.Context, srv types.Server, notifier noti.Notifier, persister persistence.ScanSnapshotPersister) {
+func HandleFolders(ctx context.Context, srv types.Server, notifier noti.Notifier, persister persistence.ScanSnapshotPersister, agg aggregator.StateAggregator) {
 	go sendFolderConfigsNotification(notifier)
-	go sendInitialSummaryPanelNotification(notifier)
+	initScanStateAggregator(agg)
 	initScanPersister(persister)
 	HandleUntrustedFolders(ctx, srv)
 }
-func sendInitialSummaryPanelNotification(notifier noti.Notifier) {
+
+func initScanStateAggregator(agg aggregator.StateAggregator) {
 	c := config.CurrentConfig()
-	logger := c.Logger().With().Str("method", "sendInitialSummaryPanelNotification").Logger()
-	renderer, err := summary.NewHtmlRenderer(c)
-	if err != nil {
-		logger.Error().Err(err).Msg("could not create html renderer")
-		return
+	var folderPaths []string
+	for _, f := range c.Workspace().Folders() {
```

Comment:
Should this be restricted to only trusted folders? Or do we plan to have a trust summary state, when a folder is not trusted? Would actually love this, to be able to trust a folder from the summary. @andygongea

Reply:
On the other hand, we could actually add the trust status to the folder, so it would be contained. Right now, we only have `unscanned` and `scanned` status.